src/api/spec/support/amqp.rb

```diff
@@ -0,0 +1,33 @@
+module AMQPHelpers
```

Comment:
Third name for the same thing now. This should be `Rabbitmq` or `Bunny`.

Reply:
fair point - I basically took this from the bunny_mock docu